developers/integrations/compute-infrastructure/replicate/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Replicate
+sidebar_position: 2
+---
+
+[Replicate](https://replicate.com/) is a platform that allows users to run machine learning models through a cloud API. They host a plethora of open-source models, including embedding and language models, which enables users to run and fine-tune them.
+
+## Replicate and Weaviate
+Weaviate does not currently have a direct integration for Replicate. To use models on Replicate, we recommend you use [LlamaIndex](https://docs.llamaindex.ai/en/stable/api_reference/llms/replicate/) or [LangChain](https://python.langchain.com/v0.2/docs/integrations/llms/replicate/) and connect it to your Weaviate vector store. 
```

Comment:
Consider:

To use models on Replicate, we use [LlamaIndex](https://docs.llamaindex.ai/en/stable/api_reference/llms/replicate/) or [LangChain](https://python.langchain.com/v0.2/docs/integrations/llms/replicate/) to connect Replicate to your data in Weaviate.

Reply:
I kept the vector store piece in there because that's how it's set up in LlamaIndex and LangChain. 

"To use models on Replicate, you need to use [LlamaIndex](https://docs.llamaindex.ai/en/stable/api_reference/llms/replicate/) or [LangChain](https://python.langchain.com/v0.2/docs/integrations/llms/replicate/) and connect it to your Weaviate vector store."